### ISSUE TITLE
Will not load

### ISSUE THREAD

**Issue by DanielCarlyleEvans** [AugurProject/augur#543]
Since the end of the reporting period I haven't been able to get the beta to fully load.  I've tried a few times.  The site sort of loads, but no markets and no real functionality  (there are 3 green dots making it look like it is loading, but it never does).  My REP is showing and it's still showing 46.53.  I expected a likely change (hopefully increase) after the completion of the reporting due to reductions to others from inaction or wrong reporting.

Looking forward to seeing what happens after the close of the first reporting period...  not sure if everyone is having this problem.  Windows 10, Chrome.  And while sometimes slow, I never had it fail to load previously.

**Comment by DanielCarlyleEvans** [comment]
OK.  Someone on Reddit posted a link directly to a specific market in Augur.  I clicked the link and the platform loaded.  I was able to interact with that market and then also load other markets.  I was thinking my loading problem was solved, but I tested further by closing the window and trying to load in a new one at Augur.net.  It won't load that way.

On the version that did load, my REP was still 46.53, which seems wrong, but I don't really know when we should expect that to redistribute.
